writing/Zotero.md

```diff
@@ -0,0 +1,20 @@
+# Zotero
+
+  * We recommend using [Zotero](https://www.zotero.org/) (a free, open-source reference management software) to manage your bibliography and generate BibTeX entries.
+  * **Note**: Zotero's default export format is not compatible with our BibTeX conventions. You must use the Better BibTeX export format (see the [Better BibTeX documentation](https://retorque.re/zotero-better-bibtex/exporting/)).
+  * [Here](resources/zotero-config.json) is a configuration file that can be imported into Zotero to set up the correct citation style and export options.
+  <!-- make a quote -->
+  * Group sharing:
+    - We use zotero groups to manage our references. New group is created for each project. The group is shared with all the members of the project.
```

Comment:
Nit: Zotero (capitalize). "New group is" => "A new group is"

Reply:
Done